lubridate.Rmd

```diff
@@ -617,18 +617,32 @@ With indicator and renamed column
 ```{r sum NIEDERSCHLAGSDAUER - condition - Tag/Stunde on an interval of a day}
 df_sum2 <- df_group %>% 
   summarise(MENGE = sum(NIEDERSCHLAGSDAUER[NIEDERSCHLAGSINDIKATOR==1])) %>% 
+  filter(MENGE> 0)
   arrange( STATIONS_ID , TAG, STUNDE)
 head(df_sum2, 24) 
 ```
 
 Now we apply the knowledge with the indicator to the selection of autumn.
-Now we examine the duration of rainfall in autumn.
+First we examine the duration of rainfall in autumn.
 ```{r sum NIEDERSCHLAGSDAUER - condition - on an interval of season autumn}
-df_sum_autmn <- df_group_autmn %>% 
-  summarise(DAUER = sum(NIEDERSCHLAGSDAUER[NIEDERSCHLAGSINDIKATOR==1])) %>% 
-  arrange( STATIONS_ID , JAHR, MONAT, TAG)
+df_sum_autmn_group <- df_group_autmn 
```

Comment:
Should it be <code>df_sum_<b>autumn</b>\_group</code> rather than <code>df_sum_<b>autmn</b>_group</code> or are you abbreviating something?